DevTools/Compiler/C++Compiler/main.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2020, Denis Campredon <deni_@hotmail.fr>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include "LibCpp/Driver.h"
+
+int main(int ac, const char** av)
+{
+    Cpp::CppCompiler::run(ac, av);
```

Comment:
Why not put that logic here? no one else will want to embed a C++ compiler with this interface - they'd fork/exec this thing otherwise?

Reply:
I put all the logic in CppCompiler (that I forgot to rename into Driver btw), so it could be called from the testsuite.
I don't know what other interface I should expose though. One with the options as parameter ?

Reply:
Whichever works I suppose, but I feel like having an interface that's basically equivalent to `execl("the_compiler", ...)` isn't very flexible.
Though I suppose there's no particular need to embed the driver in something, so it's probably a non-issue